### PR TITLE
Prep 5.6.4 release

### PR DESCRIPTION
**Pull request by jburel**
* Bump dependencies version 
* Add entry in history

### CODE REVIEW

**Comment by jburel** [comment]
Announcement will be in the spirit of https://www.openmicroscopy.org/2020/10/01/omero-5-6-3.html

**Review by sbesson — APPROVED** [review]
If the official announcement will be drafted separately, no objections to getting this to start generating the artifacts.

For the announcement, I would say:
- all considerations regarding the Django 3.2.x testing are irrelevant for consumers although it might be worth adding a note reminding people to upgrade their OMERO.web environment if they haven't as part of the upgrade
- certainly want to mention the Bio-Formats upgrade alongside the memo invalidation
- I assume we are don't necessarily want to promote the ZarrReader integration for this release. Otherwise, we might want to bump https://github.com/ome/openmicroscopy/blob/develop/etc/omero.properties#L215 to use the latest patch release immediately

**Comment by jburel** [comment]
I don't think we need to promote the zarr reader but it does not hurt to bump the version.

File: etc/omero.properties [hunk 1]
```diff
@@ -208,9 +208,9 @@ versions.omero-scripts-url=${versions.omero-pypi}
 ###
 
 ## Internal dependencies
-versions.omero-blitz=5.5.9
-versions.omero-common-test=5.5.8
-versions.omero-gateway=5.6.8
+versions.omero-blitz=5.5.10
+versions.omero-common-test=5.5.9
+versions.omero-gateway=5.6.9
 versions.omero-scripts=5.6.1
```
Comment (sbesson) [inline etc/omero.properties]: Should this also be bumped to `version.omero-scripts=5.6.2`  which includes several bug fixes - see https://github.com/ome/omero-scripts/compare/v5.6.1...v5.6.2 ?

Comment (jburel) [reply etc/omero.properties]: yes. I thought i did bump it

**Comment by jburel** [comment]
back to green

File: history.rst [hunk 2]
```diff
@@ -5,6 +5,22 @@
...
+
+This release improves and expands integration testing to handle Django 3.2.x.
+It also removes obsolete code.
+More importantly, it has been tested with:
+
+- omero-blitz 5.5.10
+- omero-gateway-java 5.6.9
+- omero-py 5.11.1
+- omero-web 5.14.0
+- omero-dropbox 5.6.2
```
Comment (sbesson) [inline history.rst:19]: Should the `omero-scripts` bump be mentioned here?

**Review by sbesson — APPROVED** [review]
One amendment, otherwise LGTM

File: history.rst (hunk 2 above)
Comment (jburel) [reply history.rst:19]: It won't hurt

**Review by sbesson — APPROVED** [review]

**Comment by jburel** [comment]
Merging and testing the build

**Pull request merged**